.travis.yml

```diff
@@ -65,6 +66,7 @@ install:
 
   # npm stuff
   - npm install
+  - npm install appium-chromedriver --chromedriver_version="2.6"
```

Comment:
why 2.6, isn't it very old? is it because there's a very old version of chrome there?

Reply:
@jlipps Yeah, right now I'm just picking random versions of chromedriver to get working on Travis. Is there an easier way to determine which CD version is needed?

Reply:
I would say just get the latest, and then if it doesn't work you should see an error message which includes the version of chrome included on travis. Then you can find the highest version of chromedriver that supports that version of chrome.

Reply:
cf. https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/web/chromedriver.md